src/main/java/seedu/address/MainApp.java

```diff
@@ -90,7 +90,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             logger.warning("Problem while reading from the file. Will be starting with an empty Notebook");
             initialData = new Notebook();
         }
-
+        int size = initialData.getLessonWeekList().get(0).asUnmodifiableObservableList().size();
```

Comment:
I think this violates the law of demeter (dont talk to strangers) but otherwise well written code. Good job!

Reply:
was supposed to be a debugging line but forgot to remove it, thanks for spotting it!!